### PR TITLE
Analyses queue counts

### LINKED ISSUE #342: Show analysis queue count in analysis list page
## Describe your idea for a new feature
Users are often asking why their analyses are taking time to run.  From their "My Analyses" page they can't see how many runs are queued in the system.  It would be nice to have a status bar somewhere to show them how many pipelines are running & queued.

## Describe how you would access this feature?
Have a small display panel on either the "My Analyses" page, or on the IRIDA dashboard.

## Additional information
N/A

### PR DESCRIPTION
**Pull request by joshsadam**
## Description of changes
Added a react component for displaying analyses queued and running counts on all analyses listing pages, the cart page, and pipeline launch page.

Screen Shot 2020-01-15 at 9 05 06 AM

## Related issue
Fixes #342 

## Checklist
Things for the developer to confirm they've done before the PR should be accepted:

* [X ] CHANGELOG.md (and UPGRADING.md if necessary) updated with information for new change.
* [X] Tests added (or description of how to test) for any new features.
~~* [ ] User documentation updated for UI or technical changes.~~

### CODE REVIEW

File: src/main/webapp/resources/js/components/AnalysesQueue.jsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    <Tag color={"blue"}>
+      <div
+        style={{
+          width: 150,
+          display: "flex",
+          justifyContent: "space-between",
+          alignContent: "center"
+        }}
+      >
+        <FontAwesomeIcon
```
Comment (tom114) [inline src/main/webapp/resources/js/components/AnalysesQueue.jsx]: I like the idea of the running man icon, but I don't think it's clear enough.  Any ideas for text replacements?  Maybe a tooltip? I might even say it looks better with no icon or text.

Comment (joshsadam) [reply src/main/webapp/resources/js/components/AnalysesQueue.jsx]: Updated documentation and add hover tooltip.

File: src/main/webapp/resources/js/components/AnalysesQueue.jsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+  useEffect(() => {
+    fetchAnalysesQueueCounts().then(data => {
+      setRunning(data.running);
+      setQueued(data.queued);
+    });
+  }, [fetchAnalysesQueueCounts]);
+
+  return (
+    <Tooltip title={i18n("AnalysesQueue.title")} placement={"bottomRight"}>
```
Comment (tom114) [inline src/main/webapp/resources/js/components/AnalysesQueue.jsx]: Can we target this on either the left or bottom center? it gives me very weird overflows depending on screen size.

File: src/test/java/ca/corefacility/bioinformatics/irida/ria/integration/components/AnalysesQueue.java [hunk 3]
```diff
@@ -0,0 +1,26 @@
+package ca.corefacility.bioinformatics.irida.ria.integration.components;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class AnalysesQueue {
```
Comment (tom114) [inline src/test/java/ca/corefacility/bioinformatics/irida/ria/integration/components/AnalysesQueue.java]: Comment on this class please

Comment (joshsadam) [reply src/test/java/ca/corefacility/bioinformatics/irida/ria/integration/components/AnalysesQueue.java]: Updated

File: src/main/webapp/resources/js/components/AnalysesQueue.jsx (hunk 2 above)
Comment (joshsadam) [reply src/main/webapp/resources/js/components/AnalysesQueue.jsx]: Fixed

**Pull request merged**